### PR TITLE
Clean up before building

### PR DESCRIPTION
**Pull request by marcjansen**
Title says it all.

Please review.

### CODE REVIEW

**Review by marcjansen — COMMENTED** [review]
Anybody up for a review? TIA

File: package.json [hunk 1]
```diff
@@ -8,6 +8,8 @@
     "geostyler-cli": "dist/index.js"
   },
   "scripts": {
+    "build:clean": "rimraf build/",
```
Comment (ahennr) [inline package.json:11]: `dist/` folder could be cleaned as well IMHO, shouldn't it?

Comment (marcjansen) [reply package.json:11]: yes, will amend

**Comment by marcjansen** [comment]
Done, @ahennr. Do you want to take another look?

**Review by KaiVolland — APPROVED** [review]
Thanks @marcjansen. Feel free to merge.

**Review by ahennr — APPROVED** [review]
plz merge.

**Comment by marcjansen** [comment]
Cheers for the reviews.

**Comment by marcjansen** [comment]
Will merge after conflicts gave been resolved...

**Pull request merged**